doc/v2/howto/rnn/recurrent_group_en.md

```diff
@@ -1,3 +1,96 @@
 # Recurrent Group Tutorial
 
-TBD
+## Overview
+
+Sequential data is a major input type dealing with natural language processing tasks.
+
+A sentence is a sequence of words. Many sentences form a paragraph further. Therefore, a paragraph can be viewed as a nested sequence with two level, where each element of the sequence is another sequence.
```

Comment:
I feel that this paragraph need to "define" the *two-level sequential data", because the next paragraph is going to talk about it.

> Sequential data could be recursive. An example of two-level recursive sequential data is that an article is composed of a sequence of sentences, and each sentence a sequence of words.

Reply:
Yeah,  Chinese document maybe apply this comment as well.